routes/web.php

```diff
@@ -22,3 +23,5 @@
 })->middleware(['auth'])->name('dashboard');
 
 require __DIR__.'/auth.php';
+
+Route::get('/wishlist', [WishlistController::class, 'wishlist']) ->name ('wish-list');
```

Comment:
i noticed on the nav template you referred to the route as "wishlist" but here you named it "wish-list". it will not work unless the names match.